src/editors/autocompleteEditor.js

```diff
@@ -15,42 +15,6 @@ import {
 } from './../helpers/dom/element';
 import HandsontableEditor from './handsontableEditor';
 
-const AutocompleteEditor = HandsontableEditor.prototype.extend();
-
-/**
- * @private
- * @editor AutocompleteEditor
- * @class AutocompleteEditor
- * @dependencies HandsontableEditor
- */
-AutocompleteEditor.prototype.init = function(...args) {
-  HandsontableEditor.prototype.init.apply(this, args);
-  this.query = null;
-  this.strippedChoices = [];
-  this.rawChoices = [];
-};
-
-AutocompleteEditor.prototype.getValue = function() {
-  const selectedValue = this.rawChoices.find((value) => {
-    const strippedValue = this.stripValueIfNeeded(value);
-
-    return strippedValue === this.TEXTAREA.value;
-  });
-
-  if (isDefined(selectedValue)) {
-    return selectedValue;
-  }
-
-  return this.TEXTAREA.value;
-};
-
-AutocompleteEditor.prototype.createElements = function(...args) {
-  HandsontableEditor.prototype.createElements.apply(this, args);
-
-  addClass(this.htContainer, 'autocompleteEditor');
-  addClass(this.htContainer, window.navigator.platform.indexOf('Mac') === -1 ? '' : 'htMacScroll');
-};
-
 let skipOne = false;
 function onBeforeKeyDown(event) {
```

Comment:
Both `onBeforeKeyDown` and `skipOne` could be pulled into the editor class, similarly to [this example in the copyPaste plugin](https://github.com/handsontable/handsontable/blob/develop/src/plugins/copyPaste/copyPaste.js#L132).